create html,css and js

karayamanemre opened the pull request:
Created HTML,CSS and JS files. 
Added all styling, functions and including responsive design. 
Added README file and MIT license.

karayamanemre committed:
create html,css and js

karayamanemre committed:
linters errors fixed

zainsadaqat commented on README.md:
```diff
@@ -2,55 +2,39 @@
 
-> Description the project.
-
+> In this project i've build an online website for an imaginary conference. 
```
Please add a description here about your project like what your Project all about 👍🏻

zainsadaqat commented on README.md:
```diff
@@ -2,55 +2,39 @@
 
 # Project Name
```
Please add a Project Title here 👍

zainsadaqat commented on README.md:
```diff
@@ -2,55 +2,39 @@
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link](https://karayamanemre.github.io/coffee-summit/)
```
Live Demo Link isn't working, Kindly fix this 👍🏻

zainsadaqat commented on README.md:
```diff
@@ -2,55 +2,39 @@
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link](https://karayamanemre.github.io/coffee-summit/)
 
```
## [Video Presentation is Missing]
1. Please record a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video should contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
4. The video contains a demo of the project, showing the pages in different screen sizes.
5. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

zainsadaqat commented on README.md:
```diff
@@ -64,12 +48,10 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
```
Please give appropriate Credit to [Cindy Shin] in "Acknowledgements" section (https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

zainsadaqat reviewed (changes requested):
# Required Changes 🔁
# Hello @karayamanemre  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [x] Please add a brief description in the Pull Request like what you have done in this Pull Request. 👍 
- [x] "About" nav-link should only highlight when it's on the About Page. Kindly fix this 👍 
<img width="778" alt="Screenshot 2022-09-13 at 4 08 25 AM" src="https://user-images.githubusercontent.com/83048208/189773720-a5c42c27-2f18-438d-b81d-719b692bf6b0.png">

_Check the comments under the review._


<img src="https://media0.giphy.com/media/QvMIzHw30MmMAqvOeo/giphy.gif"/>

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

karayamanemre committed:
update style.css

about page link color changed.
intro heading text background changed.
filter and transition added to event photos on hover.
readme file project name and acknowledgment section updated.

karayamanemre commented:
Video presentation link: https://www.loom.com/share/07f3d0a40b794b7a9b4890b0c53a1ed2

GzimAsani commented on README.md:
```diff
@@ -1,56 +1,40 @@
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link](https://karayamanemre.github.io/coffee-summit/)
```
- [x] Live link is not working yet, kindly update it since it is a project requirement

karayamanemre replied:
it will work after i merge this branch. i set the main branch as source since i will delete this branch after merge.

GzimAsani reviewed (changes requested):
### Hello @karayamanemre             :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with fixing requirements from the last reviewer, you are almost there+1: 
 
 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sumairq reviewed (approved):
Hi @karayamanemre , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
